src/jwt.cc

```diff
@@ -137,21 +137,17 @@ Status Jwt::parseFromString(const std::string& jwt) {
   return Status::Ok;
 }
 
-Status Jwt::verifyTimeConstraint(uint64_t now) const {
+Status Jwt::verifyTimeConstraint(uint64_t now, uint64_t clock_skew) const {
```

Comment:
Add a test in `src/jwt_time_test.cc` with a non-default clock skew? Just to ensure it works.

Reply:
Done